test/controllers/admin_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class AdminControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get admin_index_url
```

Comment:
@abapolov we aren't using MiniTest in the project anymore.
Please, rewrite using RSpec. By the way, should it really act like this?